pleskdistup/actions/plesk.py

```diff
@@ -297,6 +297,37 @@ def estimate_revert_time(self) -> int:
         return 0
 
 
+class EnableEnhancedSecurityMode(action.ActiveAction):
+    name: str
+
+    def __init__(
+        self,
+        name: str = "turn on the enhanced security mode in Plesk (encrypt passwords)",
+    ):
+        self.name = name
+
+    def _prepare_action(self) -> action.ActionResult:
+        util.logged_check_call([
+            "/usr/sbin/plesk", "bin", "passwords", "--encrypt",
+        ])
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        return action.ActionResult(action.ActionState.SKIPPED)
```

Comment:
Please be aware that the use of ActionState.SKIPPED is not meant for an "empty" action. Originally, it was added to skipping of `_post_action` and `_revert_action` when the action isn't needed, based on the result of `_is_required` returning false. While in this case it has no effect, other developers could deside to use SKIPPED in an empty `_prepare_action` could lead to unexpected outcomes.